README.md

```diff
@@ -52,6 +52,7 @@ List of content:
 * [P2P Validator](https://p2p.org/) A staking pool with 8% fee
 * [TheMinaPool](https://theminapool.com) A staking pool with 5% fee
 * [0base.vc pool](https://mina.0base.vc) A staking pool with 5% fee
+* [Piconbello pool](https://mina.piconbello.com) A staking pool with 0% fee
```

Comment:
Hey Piconbello, I see that this is a temporary fee. This list is not planned to be updated constantly by checking every pool. If everybody starts to do add temporary fees, then this becomes hard to keep track of which fee is correct and which not. I would prefer every pool to keep the actual fee in this list to be fair among the others and also for simplicity. 

Could you make this change? thank you 🤗

Reply:
I see while we didn't decide on the exact fee will be charging from that period. I don't feel like 3 months is can be called temporary since fees might change and we might continue with 0 for 6 months. I also send a message about something it would be nice we can clear that out